test/images/resource-consumer/README.md

```diff
@@ -0,0 +1,66 @@
+# Resource Consumer container for testing
+
+Resource Consumer is a tool for testing kubernetes autoscaling.
```

Comment:
I would start with something like: a tool which allows to generate cpu/memory utilization ...
and then write something about the reason why it was created (testing autoscaling)